pkg/deploy/api/v1beta1/types.go

```diff
@@ -207,3 +207,13 @@ type DeploymentConfigList struct {
 	v1beta3.ListMeta `json:"metadata,omitempty" yaml:"metadata,omitempty"`
 	Items            []DeploymentConfig `json:"items" yaml:"items"`
 }
+
+type RollbackSpec struct {
+	v1beta3.TypeMeta `json:",inline" yaml:",inline"`
+	// TODO: Should this come from the URL?
+	Deployment             string `json:"deployment,omitempty" yaml:"deployment,omitempty"`
+	IncludeTriggers        bool   `json:"includeTriggers,omitempty" yaml:"includeTriggers,omitempty"`
```

Comment:
Group these together under a struct - "Options" or "Spec".  Arguably we should be using "Spec" as per upstream, but i don't know that matters as much for these sorts of mapping resources (see https://github.com/openshift/origin/pull/635 for my changes to ImageRepositoryMapping)